src/main/java/org/broadinstitute/consent/http/db/DataAccessRequestDAO.java

```diff
@@ -401,21 +414,28 @@ WHERE reference_id IN (<referenceIds>)
       """)
   void archiveByReferenceIds(@BindList(value = "referenceIds", onEmpty = EmptyHandling.NULL_STRING) List<String> referenceIds);
 
+  @SqlUpdate(
+    """
+        UPDATE data_access_request
+          SET closeout_approving_so_id = :id, closeout_so_approval_timestamp = now()
+        WHERE reference_id = :referenceId
+    """)
+  void updateDarCloseoutSO(@Bind("id") Integer signingOfficialUserId, @Bind("referenceId") String referenceId);
```

Comment:
For a future implementation conversation - we're storing this on a single DAR, but a closeout approval actually applies to all DARS in a collection.

Reply:
I think it makes sense that we're storing this on the particular DAR/progress report that included the closeout - we know then which particular closeout/PR/DAR was approved. Although, it does make it more complicated to query if the DAR collection has been closed.